src/context/dispatchTypeEnum.js

```diff
@@ -0,0 +1,14 @@
+export const dispatchTypeEnum = {
+
+    DATA_FROM_SERVER: "DATA_FROM_SERVER",
+    OGGLE_WATCH_LATER: "TOGGLE_WATCH_LATER",
+    TOGGLE_LIKE: "TOGGLE_LIKE",
+    ADD_NEW_PLAYLIST: "ADD_NEW_PLAYLIST",
+    DELETE_PLAYLIST: "DELETE_PLAYLIST",
+    TOGGLE_VIDEO_IN_PLAYLIST: "TOGGLE_VIDEO_IN_PLAYLIST",
+    ADD_TO_WATCH_HISTORY: "ADD_TO_WATCH_HISTORY"
+
+}
+
+Object.freeze(dispatchTypeEnum)
+
```

Comment:
I like this file so much. Thanks for telling me about the Object.freeze method. :D ;)